test/utils/math/Math.t.sol

```diff
@@ -201,14 +201,31 @@ contract MathTest is Test {
         vm.assume(xyHi >= d);
 
         // we are outside the scope of {testMulDiv}, we expect muldiv to revert
-        try this.muldiv(x, y, d) returns (uint256) {
-            fail();
-        } catch {}
+        vm.expectRevert();
```

Comment:
Haven't tested but I think we can do:

```suggestion
        uint8 code = 0x11;
        if(d == 0) {
          code++;
        }
        vm.expectRevert(abi.encode(code));
```

Reply:
This shouldn't work according: https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3298#discussion_r1475476542